### PR TITLE
[NET-863] Add note about Cilium reqs for NPM

### PR DESCRIPTION
**Pull request by ISauve**
### What does this PR do?

Adds a small note about the Cilium requirements for NPM.

### Motivation

Certain versions of Cilium break NPM, and it's important to communicate this to customers.

### Preview

https://docs-staging.datadoghq.com/isabelle.sauve/add-cilium-req-note/network_performance_monitoring/installation/?tab=agent

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/network_performance_monitoring/installation.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ Supported platforms include:
 
 There is an exemption to the 4.4.0+ kernel requirement for [CentOS/RHEL 7.6+][2]. The [DNS Resolution][3] feature is not supported on CentOS/RHEL 7.6.
 
+Network Performance Monitoring is compatible with Cilium installations, as long as the following requirements are met:
+1) Cilium version >= 1.6, and
+2) Kernel version >= 4.19.57, >= 5.1.16, or >= 5.2
```
Comment (leeavital) [inline content/en/network_performance_monitoring/installation.md:32]: Doesn't >=4.19.57 imply >=5.1.16 and >=5.2?

Comment (kayayarai) [reply content/en/network_performance_monitoring/installation.md:32]: Depends on if you interpret the numbers as numbers or as version numbers :) For this reason, I'm going to suggest turning the >= into english words. Something like "Cilium versions later than 1.6" and "Kernel versions later than 4.19.57, 5.1.16, or 5.2"

File: content/en/network_performance_monitoring/installation.md [hunk 2]
```diff
@@ -27,6 +27,10 @@ Supported platforms include:
 
 There is an exemption to the 4.4.0+ kernel requirement for [CentOS/RHEL 7.6+][2]. The [DNS Resolution][3] feature is not supported on CentOS/RHEL 7.6.
 
+Network Performance Monitoring is compatible with Cilium installations, as long as the following requirements are met:
```
Comment (kayayarai) [inline content/en/network_performance_monitoring/installation.md:30]:
```suggestion
Network Performance Monitoring is compatible with Cilium installations, provided the following requirements are met:
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

File: content/en/network_performance_monitoring/installation.md (hunk 1 above)
Comment (ISauve) [reply content/en/network_performance_monitoring/installation.md:32]: I think you have that backwards ( >=5.2 implies >= 5.1.16 and >= 4.19.57)?
But either way, the way I understood it is that you're fine if you a newer kernel release 5.X, X>=2, but if you want to run an older release 4.19 and 5.1 are still supported as long as the subversions are >=57/16 (respectively)
I'm only guessing though, those requirements came from [here](https://docs.cilium.io/en/v1.6/install/system_requirements/#advanced-features-and-required-kernel-version) and there really isn't much more elaboration :/

Comment (leeavital) [reply content/en/network_performance_monitoring/installation.md:32]: Either way, I interpret 5.1.16 as greater than 4.19.57

Comment (ISauve) [reply content/en/network_performance_monitoring/installation.md]: I think it's supposed to mean you can run a newer kernel release 5.X, X>=2, but there's also support for 4.19.X and 5.1.X as long as X>=57/16 (respectively)

I'm only guessing though, the kernel version requirement comes from [here](https://docs.cilium.io/en/v1.6/install/system_requirements/#advanced-features-and-required-kernel-version) and there isn't much of an explanation

Comment (shlomoartzi) [reply content/en/network_performance_monitoring/installation.md]: Wdyt about shortening to 
"Network Performance Monitoring is compatible with Cilium installations v1.6+ with kernel versions later than X, Y, and Z."

File: content/en/network_performance_monitoring/installation.md [hunk 3]
```diff
@@ -27,6 +27,10 @@ Supported platforms include:
 
 There is an exemption to the 4.4.0+ kernel requirement for [CentOS/RHEL 7.6+][2]. The [DNS Resolution][3] feature is not supported on CentOS/RHEL 7.6.
 
+Network Performance Monitoring is compatible with Cilium installations, provided the following requirements are met:
+1) Cilium version later than 1.6, and
+2) Kernel version later than 4.19.57, 5.1.16, or 5.2
+
 **Note**: Datadog does not currently support Windows and macOS platforms for Network Performance Monitoring.
 
 The following provisioning systems are supported:
```
Comment (shlomoartzi) [inline content/en/network_performance_monitoring/installation.md:36]:
```suggestion
The following **provisioning systems** are supported:
```

File: content/en/network_performance_monitoring/installation.md [hunk 4]
```diff
@@ -27,6 +27,10 @@ Supported platforms include:
 
 There is an exemption to the 4.4.0+ kernel requirement for [CentOS/RHEL 7.6+][2]. The [DNS Resolution][3] feature is not supported on CentOS/RHEL 7.6.
 
+Network Performance Monitoring is compatible with Cilium installations, provided the following requirements are met:
```
Comment (shlomoartzi) [inline content/en/network_performance_monitoring/installation.md:30]:
```suggestion
Network Performance Monitoring is compatible with **Cilium** installations, provided the following requirements are met:
```
Motivation:
The number of requirements + exceptions is starting to get lengthy + hard to parse - I would bold "Cilium", "provisioning systems", and "platform" above for readability.

File: content/en/network_performance_monitoring/installation.md (hunk 1 above)
Comment (ISauve) [reply content/en/network_performance_monitoring/installation.md]: Since the kernel version requirements are a bit lengthy, I think it's easier to read when the requirements are listed separately

**Review by leeavital — APPROVED** [review]

**Comment by shlomoartzi** [comment]
lgtm!

**Review by kayayarai — APPROVED** [review]
